backend/src/backend/enums.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class StatusEnum(Enum):
+    MISSED = 0
```

Comment:
Actually, it is a string on the database. I just used enums on the code side so that queries within the code wouldn't have to use strings.
```
    status: Mapped[Optional[StatusEnum]] = mapped_column(
        Enum(
            native_enum=False,
            validate_strings=True,
            create_constraint=True,
            name="status",
        )
    )
```
The option `native_enum=False` ensures that is not an enum on the database but rather a string which is constrained to a set of values. What do you think? Should I remove it regardless?

Reply:
Perfect then